onnx_chainer/functions/math.py

```diff
@@ -48,7 +48,8 @@ def convert_Mul(func, onnx_op_name, opset_version, input_names, output_names,
 def convert_MulConstant(func, onnx_op_name, opset_version, input_names,
                         output_names, parameters):
     value = np.asarray([func.value], dtype=func.inputs[0].dtype)
-    value = np.broadcast_to(value, func.inputs[0].shape)
+    if func.inputs[0].shape:
+        value = np.broadcast_to(value, func.inputs[0].shape)
```

Comment:
JFYI: This is unnecessary and probably inefficient for opset>=7 since ONNX adds broadcast to binary ops.

Reply:
I confirmed all tests succeed when this broadcast process is removed on my local, had better to consider about legacy opset ret https://github.com/chainer/onnx-chainer/pull/63#pullrequestreview-200541636

Reply:
remove this broadcast process. Testing under opset>=7, all test succeed.